src/css/custom.css

```diff
@@ -69,6 +69,26 @@ main {
   margin-top: 2em;
 }
 
+/* IMAGES & CAPTIONS */
+
+p > img {
+  display: block;
+  margin: auto;
+}
+
+img + em {
```

Comment:
I wasn't familiar with this selector - what does it mean to have `img + em` is it: 
```
<em><img /> </em>
```?

Reply:
[Adjacent sibling combinator](https://css-tricks.com/child-and-sibling-selectors/#adjacent-sibling-combinator). 👍